### PR TITLE
Scavs: Point system updates

### PR DESCRIPTION
**Pull request by whalemer**
@CheeseMuffin

### CODE REVIEW

File: chat-plugins/scavengers.js [hunk 1]
```diff
@@ -1157,43 +1177,65 @@ let commands = {
...
 		if (!this.can('mute', null, room)) return false; // perms for viewing only
-		if (!target) return this.sendReply(`The points rewarded for winning official hunts is: ${(room.blitzPoints || DEFAULT_BLITZ_POINTS)}`);
+
+		if (!target) {
+			let points = [];
+			const source = Object.entries(Object.assign(DEFAULT_BLITZ_POINTS, room.blitzPoints || {}));
+			for (const entry of source) {
+				points.push(`${entry[0]}: ${entry[1]}`);
+			}
+			return this.sendReplyBox(`The points rewarded for winning hunts is:<br />${points.join('<br />')}`);
```
Comment (CheeseMuffin) [inline chat-plugins/scavengers.js]: The message here should be about blitz, right?

File: chat-plugins/scavengers.js [hunk 2]
```diff
@@ -807,12 +823,13 @@ let commands = {
 	 */
 	createpractice: 'create',
 	createofficial: 'create',
+	createunrated: 'create',
```
Comment (CheeseMuffin) [inline chat-plugins/scavengers.js]: Is this for creating hunts that won't give out any points?

File: chat-plugins/scavengers.js [hunk 3]
```diff
@@ -1157,43 +1177,65 @@ let commands = {
...
+				points.push(`${entry[0]}: ${entry[1].map((p, i) => `(${(i + 1)}) ${p}`).join(', ')}`);
+			}
+			return this.sendReplyBox(`The points rewarded for winning hunts is:<br />${points.join('<br />')}`);
+		}
 
 		if (!this.can('declare', null, room)) return false; // perms for editting
 
-		let winPoints = target.split(',').map(p => parseInt(p));
+		let [type, ...pointsSet] = target.split(',');
+		type = toId(type);
```
Comment (CheeseMuffin) [inline chat-plugins/scavengers.js]: Should we check that `type` is something we actually want to set (e.g. official or regular)? So we don't accidentally start setting the points for practice/unrated.

File: chat-plugins/scavengers.js (hunk 2 above)
Comment (whalemer) [reply chat-plugins/scavengers.js]: yes, requested by a ro

**Comment by CheeseMuffin** [comment]
Looks good now, only thing is would it be possible to have one more rated type, "mini" (or "miniofficial") (and a createmini command)? We've been doing them as less intense version of regular officials, doesn't need default points or anything.

**Comment by whalemer** [comment]
updated

**Pull request merged**